Gaijinizer patch/CommonEvents_128.txt

```diff
@@ -86,31 +86,39 @@ them as much as you want...♡#Triketos
 
 甘えん坊な養分くんだなぁ……♡
 > CONTEXT: CommonEvents_128/143/Dialogue/enemy
-Nain ぃ ♡. Sweethearted Nutrient-kun nah...♡#MTLed
+Whaat?♡
+
+Spoiled little Nutrients-Kun...♡#Strate
 > END STRING
```

Comment:
Took a look at `Nutrients-Kun`... we've have different versions of capitalizations of it everywhere. -_-

I'm going to standaradize all of them to `Nutrients-kun`.